src/template.html

```diff
@@ -158,6 +158,7 @@
                 {{sesameVersion}} &bull; Connectors {{connectorsVersion}} &bull; Workbench {{workbenchVersion}} &bull; &copy; {{previousYear}}&ndash;{{currentYear}} <a href="http://ontotext.com" target="_blank" rel="noopener noreferrer">Ontotext AD</a>. {{'all.rights.reserved.label' | translate}}
             </p>
         </div>
+        <cookie-consent></cookie-consent>
```

Comment:
Not sure if this should be here or in the user template where the actual link which opens the colicy dialog

Reply:
should be able to display on every page - this is the consent popup